components/taghints.py

```diff
@@ -101,7 +101,9 @@
     },
     "wronglib": {
         "message": (
-            "{query} If you insist on using that other one, please go where you belong: "
+            "{query} : If you are are using a different package/language, we are sure you can "
+            "find some kind of community help on their homepage. Here are a few links for other"
+            "popular libraries: "
             '<a href="https://telegram.me/joinchat/Bn4ixj84FIZVkwhk2jag6A">pyTelegramBotApi</a>, '
             '<a href="https://github.com/nickoala/telepot">Telepot</a>, '
             '<a href="https://t.me/Pyrogram">pyrogram</a>, '
```

Comment:
```suggestion
            '<a href="https://t.me/pyrogramchat">pyrogram</a>, '
```
we should link to the chat like the others